### PR TITLE
Meeting

### PR DESCRIPTION
**Pull request by HeloiseFerla**

### CODE REVIEW

**Review by Quentin-Macquart — DISMISSED** [review]

File: src/pages/Meeting/Room/index.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                        {slot.description}
+                      </button>
+                    );
+                  })}
+                </div>
+              </SwiperSlide>
+            );
+          })}
+        </Swiper>
+        {renderModal ? (
```
Comment (Hacene010) [inline src/pages/Meeting/Room/index.jsx:56]: You can do the same thing with renderModal && ...

File: src/pages/Meeting/Room/style.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+const SRoom = styled.main`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  h2 {
+    margin: 15px;
+    font-size: 1.2rem;
+    font-weight: bold;
+    justify-self: left;
```
Comment (Hacene010) [inline src/pages/Meeting/Room/style.jsx:12]: is this working ? I thought it's flex-start ??

File: src/pages/Meeting/RoomCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  equipment,
+  picture,
+}) {
+  return (
+    <SRoomCard>
+      <h2>Salle de reunion n°{id}</h2>
+      <p>{localisation}</p>
+      <p>Capacité : {capacity} </p>
+      <p>{equipment}</p>
+      <img src={picture} alt="" />
```
Comment (Hacene010) [inline src/pages/Meeting/RoomCard/index.jsx:17]: You forgot the alt here. 
And plz talk with the others to put the same alt langage

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to make !

**Pull request closed without merging**